src/app/charts/chart.component.html

```diff
@@ -65,15 +65,30 @@ <h2 class="chart-label">
                                 <h4>Get API Call</h4>
                                 <p>Use our API to access these data regularly for your own research or application.</p>
                                 <div class="input-group">
+                                    <span class="curl-label">Historical data</span>
```

Comment:
It would be nice to see this `div.input-group` DRYed into a separate `CopyCurlComponent` or something of that nature. There are clear inputs/outputs here with a DOM/functionality that is almost exactly the same -- only the value of the input and its placeholder changes between the two.